packages/forms/src/UIForm/Widget/Widget.component.js

```diff
@@ -1,15 +1,27 @@
 import PropTypes from 'prop-types';
 import React from 'react';
+import get from 'lodash/get';
+import includes from 'lodash/includes';
 import { sfPath } from 'talend-json-schema-form-core';
 
 import defaultWidgets from '../utils/widgets';
 import { getValue } from '../utils/properties';
 
+function shouldRender(conditions, properties) {
+	return (
+		!conditions ||
+		conditions.every(cond => {
+			const value = get(properties, cond.path);
```

Comment:
No need for that constant. It's only used once.

Reply:
Thanks, removed